apps/framework-cli/src/cli/routines.rs

```diff
@@ -257,16 +258,87 @@ pub async fn start_development_mode(project: Arc<Project>) -> anyhow::Result<()>
     Ok(())
 }
 
+// Starts the webserver in production mode
+pub async fn start_production_mode(project: Arc<Project>) -> anyhow::Result<()> {
+    show_message!(
+        MessageType::Success,
+        Message {
+            action: "Starting".to_string(),
+            details: "production mode".to_string(),
+        }
+    );
+
+    let mut route_table = HashMap::<PathBuf, RouteMeta>::new();
+
+    info!("Initializing project state");
+    initialize_project_state(project.clone(), &mut route_table).await?;
+    let route_table: &'static RwLock<HashMap<PathBuf, RouteMeta>> =
```

Comment:
Do we actually need the lock here? Might be something to look into as we explore the production server going forward

Reply:
Yeah I think this can be just a reference, but I don't mind merge this as is.